prowler/providers/aws/services/workspaces/workspaces_service.py

```diff
@@ -45,11 +45,13 @@ def __describe_workspaces__(self, regional_client):
                         )
                     ):
                         workspace_to_append = WorkSpace(
-                            id=workspace["WorkspaceId"], region=regional_client.region
+                            id=workspace.get("WorkspaceId"),
+                            region=regional_client.region,
+                            subnet_id=workspace.get("SubnetId"),
```

Comment:
Testing it I get an error when the Workspace is in `PENDING` state because it doesn't have a `SubnetID`. Could we control that?

Should we check the status of each workspace?

```
2023-04-27 10:44:14,650 [File: workspaces_service.py:66]        [Module: workspaces_service]     ERROR: eu-west-1 -- KeyError[51]: 'SubnetId'
```